src/System.IO.Compression.ZipFile/tests/ZipFileConvenienceMethods.cs

```diff
@@ -15,10 +16,13 @@ public partial class ZipTest
         public async Task CreateFromDirectoryNormal()
         {
             await TestCreateDirectory(zfolder("normal"), true);
-            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows)) // [ActiveIssue(5459, PlatformID.AnyUnix)]
-            {
-                await TestCreateDirectory(zfolder("unicode"), true);
-            }
+        }
+
+        [Fact]
+        [Trait(XunitConstants.Category, XunitConstants.IgnoreForCI)]
```

Comment:
Can you add a brief comment next to these citing why they're being ignored for CI?